tests/utils.py

```diff
@@ -65,3 +66,35 @@ def assert_log_contains(
 
     assert all(inc in messages for inc in includes)
     assert all(exc not in messages for exc in excludes)
+
+
+def decode_ansi(output: str) -> str:
```

Comment:
I wouldn't say it's "decode"ing the ansi, I'd call it "strip_ansi_codes" or similar

Reply:
I meant it as decoding strings with ansi in them, but yeah that's probably clearer